<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1164</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SCU URP Helper</string>
  </property>
  <widget class="QTableWidget" name="table">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>1101</width>
     <height>531</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="query_info">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="query_btn">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>20</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>输入课程名称：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_btn">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>20</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
